src/libzippp.h

```diff
@@ -127,15 +127,41 @@ namespace libzippp {
             ORIGINAL,
             CURRENT
         };
+        /**
+         * Defines encryption methods to be used, when writing, by the underlying libzip library.
+         * These algorithms map to the types defined in libzip,
+         * with the addition of a "ZIP_" prefix. For details see:
+         * https://libzip.org/documentation/zip_file_set_encryption.html
+         * NONE will use no encryption.
+         * AES_128 will use Winzip AES-128 encryption.
+         * AES_192 will use Winzip AES-192 encryption.
+         * AES_256 will use Winzip AES-256 encryption.
+         * TRAD_PKWARE will use  Traditional PKWare encryption. Do not use this method, it is not secure. It is only provided for backwards compatibility.
+         */
+        enum Encryption{
+            NONE,
+#ifdef LIBZIPPP_WITH_ENCRYPTION
+            AES_128,
+            AES_192,
+            AES_256,
+            TRAD_PKWARE
+#endif
```

Comment:
In prevision of #87 to solve #54, please use CamelCase for enums: `None,Aes128,Aes192,Aes256,TradPkware`.